client/app.jsx

```diff
@@ -1,40 +1,67 @@
-import React/* { useState, useEffect } */from 'react';
-import Week from './pages/day';
+import React, { useState, useEffect } from 'react';
+import Schedule from './components/schedule';
 import Header from './components/header';
-import PageView from './pages/page-content';
+import PageContent from './components/page-content';
 import Navbar from './components/navbar';
-// import { parseRoute } from './lib';
-
-// export default class App extends React.Component {
-//   render() {
-//     return <Home />;
-//   }
-// }
+import Planner from './pages/planner';
+import { Calories } from './pages/calories';
+import { Stopwatch } from './pages/stopwatch';
+import { AddWorkout } from './pages/add-workout';
+import { parseRoute } from './lib';
 
 const App = () => {
-  // const [route, setRoute] = useState(parseRoute(window.location.hash));
+  const [route, setRoute] = useState(parseRoute(window.location.hash));
 
-  // useEffect(() => {
-  //   window.addEventListener('hashchange', event => {
-  //     setRoute(parseRoute(window.location.hash));
-  //   });
-  // });
+  useEffect(() => {
+    window.addEventListener('hashchange', event => {
+      setRoute(parseRoute(window.location.hash));
+    });
+  });
 
-  // const renderPage = () => {
-  //   if (route.path === 'page1') {
-  //     return <Home />;
-  //   }
-  //   if (route.path === 'page2') {
-  //     return <CurrentDay />;
-  //   }
-  // };
+  const renderPage = () => {
+    if (route.path === '') {
+      return <Planner />;
+    }
+    if (route.path === '1') {
+      return <Planner />;
+    }
+    if (route.path === '2') {
+      return <Planner />;
+    }
+    if (route.path === '3') {
+      return <Planner />;
+    }
+    if (route.path === '4') {
+      return <Planner />;
+    }
+    if (route.path === '5') {
+      return <Planner />;
+    }
+    if (route.path === '6') {
+      return <Planner />;
+    }
+    if (route.path === '7') {
+      return <Planner />;
+    }
```

Comment:
Since these are all returning the same component you can simplify this with something like:
```JAVASCRIPT
if (route.path <= '7') {
  return <Planner />
}
```

Not sure that exact `if` condition will work but something along those lines should.

Reply:
Thank you for some reason, since it was a number string i didnt even think about doing that. It may work, i want to display specific lists for the each different day from the database. So i need to come up with a way to display the specific page once the day is clicked.

Reply:
Ultimately you can pass the route info as a prop so inside of `<Planner />` you can use that information to get the correct information for that day. It would look something like:
```JAVASCRIPT
if (route.path <= '7') {
  return <Planner day={route.path} />
}
```